Events/services.js

```diff
@@ -0,0 +1,44 @@
+// This function return a json containing the eventuid of the newly created event.
+var createEvent = function(req) {
+  console.log(req);
+
+  // TODO (zachlefevre): Write method and remove console.log().
+  return req; // This will be replaced with a json containing the eventuid of the newly created event.
+}
+
+// This function returns a json containing the event represented by the eventuid.
+var getEventByID = function(req) {
+  console.log(req);
+  // TODO (zachlefevre): Write method and remove console.log.
+  return {
+    "event": {
+      "category" : "ENUM/STRING",
+      "contact" : "STRING",
+      "creator" : "USERUID",
+      "datetime" : "DATE-TIME STRING",
+      "description" : "STRING",
+      "location" : "STRING",
+      "title" : "STRING"
+}
+    }; // This will be replaced with a json containing the event represented by the eventuid.
+}
+
+// This function deletes the event represented by the eventuid.
+var deleteEvent = function(req) {
+  console.log(req);
+  // TODO (zachlefevre): Write method and remove console.log().
+  return req; // This will be removed.
+}
+
+// Returns a json containing every event.
+var getAllEvents = function() {
+  console.log("getAllEvents");
+  // TODO (zachlefevre): Write method and remove console.log().
+  return {
```

Comment:
Don't forget to add the status when you return. The router doesn't know about the success or failure of the call.

```javascript
return {
  status: 200,
  object: {
    "1234": { /* a fake event object */},
    "4312": { /* a different fake event object */ }
  }
};
```